mentat/code_context.py

```diff
@@ -42,6 +43,7 @@ class CodeContextSettings:
     diff: Optional[str] = None
     pr_diff: Optional[str] = None
     no_code_map: bool = False
+    no_embedding: bool = True
```

Comment:
I think this would make more sense (along with no_code_map) to be just embedding (or has_embeddings) and have it default to False